### DISCUSSION TITLE
Wavepack conversion not producing correction file

### DISCUSSION THREAD

**Discussion by zjDrummond** [enzo1982/freac#658] [Help & Support]
I wanted to try out wavepack after I heard that it can do hybrid lossy/lossless compression, but after I tried it out the only output file I'm getting is the song.wv file. I'm missing the .wvc file. I have enabled create correction file in the wavepack encoder settings. 

I'm running fre:ac v1.1.7 flatpak on Kubuntu 24.04. Please send help. Thanks.

**Comment by glanduin** [comment]
I have tried this recently and i actually searched and found the correction file in the "temp" folder (/**_username_**/.var/app/org.freac.freac/cache/tmp/). So the correction file (wvc) is actually created but it is left in the temp folder and not carried over. It could be better if both the lossywav files and the correction files were put in the same folder, or maybe a setting in the wavpack encoder options to point to a location for the correction files only.

**Comment by ikyquocked** [comment]
Proved bug. No correction file in the output folder after converting wavpack -c
